config/config.go

```diff
@@ -31,6 +31,8 @@ type Config struct {
 
 	ExcludedReasons []string `yaml:"excluded.reasons,flow"`
 	IncludedReasons []string `yaml:"included.reasons,flow"`
+
+	IncludedNamespace []string `yaml:"included.namespaces,flow"`
```

Comment:
don't forget to changes this file https://github.com/warungpintar/siera-kube-watch/blob/master/deployment/config.yaml

Reply:
Done!